src/HubPortal.sol

```diff
@@ -29,7 +29,10 @@ contract HubPortal is Portal, IHubPortal {
     uint128 public disableEarningIndex;
 
     /// @inheritdoc IHubPortal
-    mapping(uint256 destinationChainId => uint256 principal) public bridgedPrincipal;
+    mapping(uint256 spokeChainId => uint256 principal) public bridgedPrincipal;
```

Comment:
review if introducing struct and packing bool + principal fields into 1 storage slot justified extra efforts and reducing readability.